extensions/podman/src/podman-cli.ts

```diff
@@ -111,20 +111,13 @@ export function execPromise(
     process.stderr.setEncoding('utf8');
     process.stderr.on('data', data => {
       stdErr += data;
-      options?.logger?.error(data);
+      options?.logger?.warn(data);
     });
 
     process.on('close', exitCode => {
-      let content = '';
-      if (stdOut && stdOut !== '') {
-        content += stdOut + '\n';
-      }
-      if (stdErr && stdErr !== '') {
-        content += stdErr + '\n';
-      }
-
       if (exitCode !== 0) {
-        reject(new Error(content));
+        options?.logger?.error(stdErr);
```

Comment:
Same data will be logged twice. Is it a requirement ?

Reply:
WDYM logged twice? the logger.error and the reject message? or the warn and the error?
if it is about the first, then the reject will be handled by a catch in further steps so it's the ui which will decide if to show it twice somewhere or not. In the current implementation it happens as you can read the error in the terminal and in the error component that pop up below the terminal.

if it's about the second, the task manager uses the error content as the text to show if an error occurs. But it seems correct for me. Let's say i only write to the warn channel, the task manager should have the logic to store the content and then show it in the task manager UI if the process fails. 

![image](https://user-images.githubusercontent.com/49404737/233076349-44aa57ea-9907-48db-913c-e0840642eeba.png)

By maintaining this separation the UI can prints the warn in the terminal and the error (if the process eventually fails) in the task manager.

Reply:
I mean the content of stdErr will be logged twice at the warn and error level, don't know if it's an issue ?

Reply:
forgot to reply. The problem is that you don't know in advance if the content returned by stderr is just a warning or a fatal error that will stop the process. Only when the process fails you can tell, ok the stderr returned was a fatal error. I don't see many other alternatives how to handle it tbh.
The decision is just if we want to redirect stderr to logger.log or logger.warn.
If you have any preferences, just let me know so i can update it.